src/store/card.js

```diff
@@ -58,6 +58,10 @@ export default {
 					}
 				}
 
+				if (unassigned) {
+					return card.assignedUsers.length === 0
```

Comment:
This will not work when both unassigned and a due filter are set since it returns to early